.github/workflows/release.yml

```diff
@@ -74,6 +74,13 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           gh release delete ${{ steps.next_version.outputs.version_tag }} || true
+      - name: Set Package Version
+        run: |
+          cat << EOF > opslevel/version.go
+          package opslevel
+
+          const version string = "${{ steps.next_version.outputs.version_tag }}"
+          EOF
```

Comment:
We don't need to do this anymore once the ldflags are fixed

Reply:
Ahhhh good call